### PR TITLE
docs: Tweak docs for 0.4.0

### PR DESCRIPTION
**Pull request by richb-hanover**
A few documentation changes to reflect a) my better understanding of PRQL, and b) some edits to handle 0.4.0

### CODE REVIEW

File: book/src/queries/syntax.md [hunk 1]
```diff
@@ -94,9 +94,25 @@ from employees
 select first_name
 ```
 
-## Parentheses
+## Expressions
 
-Parentheses — `()` — are used to give precedence to inner expressions.
+Many PRQL statements compute a value with an _expression_. Expressions follow
+the normal rules for computer languages. In the example below, note the use of
```
Comment (max-sixty) [inline book/src/queries/syntax.md]:
```suggestion
PRQL is made up of  _expressions_, like `2 + 3` or `(1 + x) * y)`. In the example below, note the use of
```

WDYT?

File: book/src/queries/pipelines.md [hunk 2]
```diff
@@ -42,4 +43,6 @@ select [e.name, gross_salary, d.name]
 
 ## See also
 
-[Syntax](./syntax.md)
+[Transforms](../transforms.html) - PRQL Transforms
+
+[Syntax](./syntax.md) - Notation for PRQL queries
```
Comment (max-sixty) [inline book/src/queries/pipelines.md:48]:
```suggestion
[Syntax](./syntax.md) - Notation for PRQL queries
```

**Review by max-sixty — COMMENTED** [review]
These are great @richb-hanover — thank you!

The tests are failing since the examples are tested. Running `task test-rust` will set the snapshots, or I can do it if you don't have a dev environment going

File: book/src/queries/syntax.md (hunk 1 above)
Comment (richb-hanover) [reply book/src/queries/syntax.md]: Yes, but the second expression needs an opening "("

Then LGTM

**Comment by richb-hanover** [comment]
I'm letting my Dockerfile chew on `task test-rust` now...

**Comment by richb-hanover** [comment]
@max-sixty - I'm going to ask you to run `task test-rust`. It works on my Docker container... I'l check the diff to see what went awry. Thanks.

**Comment by max-sixty** [comment]
`test-rust` passes...

Thank you @richb-hanover ! Good improvements.

**Pull request merged**